src/pybito.cpp

```diff
@@ -934,34 +974,39 @@ PYBIND11_MODULE(bito, m) {
            "Get scores from NNIs from previous iterations.")
       .def("iter_count", &NNIEngine::GetIterationCount,
            "Get number of iterations of NNI search run.")
+      .def("nnis_to_rescore", &NNIEngine::GetNNIsToRescore,
```

Comment:
Let's move past_scored_nnis and nnis_to_rescore up a few lines to the access section and add the relevant count functions here. Also let's add the count function for the scored nnis.

Reply:
Moved functions and added count functions.